dashboard/app/models/script.rb

```diff
@@ -608,11 +598,37 @@ def csf_finish_url
   end
 
   def summarize
+    summarized_stages = stages.map(&:summarize)
+
+    if peer_reviews_to_complete
+      levels = []
+      peer_reviews_to_complete.times do |x|
+        levels << {
+            ids: [x],
+            position: 0,
```

Comment:
We don't use it with dots, but I don't know where else we are using it right now. I'll remove it here but won't touch it in scriptlevel until I've understood that part more